PermanentOrg/web-app
# Per 9228 create goal setting flow

This pull request serves as the implementation of the new onboarding flow. I have opted to make one pr with all the changes instead of 3 pull requests, mainly because I would get confused when having to change things in all 3 of them if changes were needed. 

Steps to test: 

Without pending archive invitations: 

1.Create a new account using a valid email for mail chimp
2.Input all the needed informations and tick the corresponding squares
3. Open the network tab before creating the archive
4. After creating the archive, check the network tab, there should be an API call with a "tags" name that sends everything to mail chimp.

With pending archive invitations everything works as before, but the button that allowed you to create an archive instead of accepting the invitation takes you now to the new onboarding screen.
